generatorv2/lib/utils.rb

```diff
@@ -47,7 +47,7 @@ def additional_json(json)
   def remove_tests(uuid, json)
     json["cases"].each_with_object([]) do |x, acc|
       if x["cases"]
```

Comment:
```suggestion
      acc << if x["cases"]
```

This, along with ...